CHANGELOG.md

```diff
@@ -2,193 +2,182 @@
-  - lib moved into gems/pending
-  - Build and system directories extracted to new repositories
-
-
-### [Removed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+26+Ending+July+13%2C+2015%22+label%3A%22technical+debt%22)
-
-- Core: SOAP server side has been removed
-
-## Sprint 25 end 2015-06-22
-
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+25+Ending+June+22%2C+2015%22+label%3Aenhancement)
-
-- Container Integration: Kubernetes Events
-- Amazon: Added M4 and t2.large instance types
-- UI: Added donut charts
-
-### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+25+Ending+June+22%2C+2015%22+label%3Aenhancement)
-
-- Automate: Automate Server Role enabled by default
-- Service Dialogs: Auto-refresh for dynamic dialog fields
-- UI: Upgraded to jQuery 1.9.1
-- UI: Increased chart responsiveness
-- Appliance: Extracting C code to external gems
-  - MiqLargeFileLinux => [large\_file\_linux]( https://github.com/ManageIQ/large_file_linux) gem
-  - MiqBlockDevOps => [linux\_block\_device](https://github.com/ManageIQ/linux_block_device) and [memory\_buffer](https://github.com/ManageIQ/memory_buffer) gems
-  - MiqMemory => [memory\_buffer](https://github.com/ManageIQ/memory_buffer) gem
-  - SlpLib => [slp](https://github.com/ManageIQ/slp) gem
-
-## Sprint 24 end 2015-06-01
-
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+24+Ending+June+1%2C+2015%22+label%3Aenhancement)
-
-- Kubernetes inventory
-  - Replicators
-  - Routes
-  - Projects
-- OpenShift Provider
-- OpenStack: Better name for AMQP binding queues
-- Provisioning
-  - Enabled SCVMM Auto placement
-  - Provision dialog field validation for non-required fields
-
-### [Fixed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+24+Ending+June+1%2C+2015%22+label%3Abug)
-
-Notable fixes include:
-- Provisioning: Fixed unique names for provisioned VMs when ordered through a service
-- UI
-  - Fixes for Japanese I18n support
-  - Fixed alignment of Foreman explorer RBAC features with the UI
 
-## Sprint 23 end 2015-05-11
+### SmartState
+- SmartState Analysis
+ - Support for VMware VDDK version 6
 
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+23+Ending+May+11%2C+2015%22+label%3Aenhancement)
+### Security
+- Authentication
+ - External Auth to AD web ui login & SSO - Manual configuration
+ - External Auth to LDAP - Manual configuration
 
-- Foreman
-  - Filtering of available Configuration Profiles based on selected Configured Systems in provisioning dialog
-  - Exposed additional properties to reporting
+### Appliance
+- Supporting Additional External Authentications
+- 2-Factor Authentication with FreeIPA >= 4.1.0
+- Nightly builds (Mon-Fri) on http://releases.manageiq.org
+- PostgreSQL 9.2 => 9.4.1
+- Changed file system from ext4 to xfs  
+- Added support for systemctl
+- CentOS 7.1
+- Apache 2.4
+- thin => puma
```

Comment:
Ugh.  Nevermind.  Reread what you just said.  I'll fix it up.


On Thu, Sep 24, 2015 at 11:35 AM, Marianne Feifer <mfeifer@redhat.com>
wrote:

> I'll fix that, but it was reported in a Sprint Review Meeting.  Is there a
> different way it should be presented?
>
> On Thu, Sep 24, 2015 at 11:15 AM, Matthew Draper <notifications@github.com
> > wrote:
>
>> In CHANGELOG.md
>> <https://github.com/ManageIQ/manageiq/pull/4407#discussion_r40330788>:
>>
>> >
>> > -- Foreman
>> > -  - Filtering of available Configuration Profiles based on selected Configured Systems in provisioning dialog
>> > -  - Exposed additional properties to reporting
>> > +### Appliance
>> > +- Supporting Additional External Authentications
>> > +- 2-Factor Authentication with FreeIPA >= 4.1.0
>> > +- Nightly builds (Mon-Fri) on http://releases.manageiq.org
>> > +- PostgreSQL 9.2 => 9.4.1
>> > +- Changed file system from ext4 to xfs
>> > +- Added support for systemctl
>> > +- CentOS 7.1
>> > +- Apache 2.4
>> > +- thin => puma
>>
>> This isn't true... while we now support running on puma, the appliance
>> still defaults to thin
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/ManageIQ/manageiq/pull/4407/files#r40330788>.
>>
>
>
>
> --
> Marianne Feifer
> Senior Technical Project Manager
> CloudForms Management Engine
> mfeifer@redhat.com
> irc: mfeifer
> freenode: marigold
>



-- 
Marianne Feifer
Senior Technical Project Manager
CloudForms Management Engine
mfeifer@redhat.com
irc: mfeifer
freenode: marigold